app/domain/ca/certificate_authority.rb

```diff
@@ -55,13 +54,52 @@ def sign_csr(csr, ttl)
         extension_factory.create_extension('subjectKeyIdentifier', 'hash')
       )
 
+      csr_cert.add_extension(
+        extension_factory.create_extension("subjectAltName", subject_alt_name(role))
+      )
+
       csr_cert.sign private_key, OpenSSL::Digest::SHA256.new
-  
       csr_cert
     end
 
     protected
 
+    def subject(role)
+      common_name = [
+        'conjur', 
```

Comment:
I'm ok with that. I think we had settled on this because it roughly mirrored the SPIFFE ID, but there is no other reason I am aware of at this point. I will change it.